volatility3/framework/plugins/windows/sessions.py

```diff
@@ -0,0 +1,96 @@
+# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import datetime
+import logging
+
+from volatility3.framework import renderers, interfaces
+from volatility3.framework.configuration import requirements
+from volatility3.framework.objects import utility
+from volatility3.plugins.windows import pslist
+
+vollog = logging.getLogger(__name__)
+
+
+class Sessions(interfaces.plugins.PluginInterface):
+    """lists Processes with Session information extracted from Environmental Variables"""
+
+    _required_framework_version = (2, 0, 0)
+
+    @classmethod
+    def get_requirements(cls):
+        return [
+            requirements.TranslationLayerRequirement(name = 'primary',
```

Comment:
Ideally this would use the new `ModuleRequirement` (which essentially just wraps the translation layer and symbol table, but it also holds information such as where the module lives, etc).

Reply:
That should be good now.